src/foundry/common/constants.d.mts

```diff
@@ -876,6 +899,8 @@ export declare const TABLE_RESULT_TYPES: Readonly<{
 
   /**
    * A Compendium Pack reference which will be linked to in the chat message.
+   * @deprecated "`CONST.TABLE_RESULT_TYPES.COMPENDIUM` is is deprecated in favor of {@linkcode CONST.TABLE_RESULT_TYPES.DOCUMENT}
+   * due to the "compendium" being merged with the "document" type." (since v13, until v15)
```

Comment:
Can we put this on one line? I know it's awkwardly long but, y'know, ESLint again.